subsys/spm/Kconfig

```diff
@@ -34,6 +34,7 @@ menuconfig IS_SPM
 	bool "Current app is SPM"
 	default n
 	select TRUSTED_EXECUTION_SECURE
+	# depends on !FPU
```

Comment:
I would assume this is supposed to be included?

Reply:
Yes, but it gives a Kconfig dependency loop that I couldn't figure out.

Reply:
Do you actually need the dependency? I'd vote to not list it at all.

Btw, you can clean the default n, as well.

Reply:
I see this was merged with:
https://github.com/nrfconnect/sdk-nrf/blob/620f42ac361877a9d15c81df320300e4b8a76a43/subsys/spm/Kconfig#L38

@oyvindronningstad can you clean it up ?

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/4958